src/Contracts/index.ts

```diff
@@ -0,0 +1,6 @@
+/**
+ * Shape of the SpecReporter possible options
+ */
+export interface SpecReporterOptions {
+  stackLinesCount: number
```

Comment:
Can we make this optional as well. Right now there is only one option, but if we add more in the future, we should not force defining `stackLinesCount` alongside with other options